lib/src/executables/intl_message_migration.dart

```diff
@@ -0,0 +1,315 @@
+  // args, but that's not as convenient to the user and makes showing help a bit more complicated.
+  final codemodArgs = _allCodemodFlags
+      .where((name) => parsedArgs[name] as bool)
+      .map((name) => '--$name')
+      .toList();
+
+  // codemod sets up a global logging handler that forwards to the console, and
+  // we want that set up before we do other non-codemodd things that might log.
+  //
+  // We could set up logging too, but we can't disable codemod's log handler,
+  // so we'd have to disable our own logging before calling into codemod.
+  // While hackier, this is easier.
+  // TODO each time we call runInteractiveCodemod, all subsequent logs are forwarded to the console an extra time. Update codemod package to prevent this (maybe a flag to disable automatic global logging?)
+  exitCode = await runInteractiveCodemod(
+    [],
+    (_) async* {},
+    args: codemodArgs,
+    additionalHelpOutput: parser.usage,
+  );
+  if (exitCode != 0) return;
+  print('^ Ignore the "codemod found no files" warning above for now.');
+
+  /// Runs a set of codemod sequences separately to work around an issue where
+  /// updates from an earlier suggestor aren't reflected in the resolved AST
+  /// for later suggestors.
+  ///
+  /// This means we have to set up analysis contexts multiple times, which takes longer,
+  /// but isn't a dealbreaker. E.g., for wdesk_sdk, running two sequences takes 2:52
+  /// as opposed to 2:00 for one sequence.
+  ///
+  /// If any sequence fails, returns that exit code and short-circuits the other
+  /// sequences.
+
+  Future<int> runCodemodSequences(
+    Iterable<String> paths,
+    Iterable<Iterable<Suggestor>> sequences,
+  ) async {
+    for (final sequence in sequences) {
+      final exitCode = await runInteractiveCodemodSequence(
+        paths,
+        sequence,
+        defaultYes: true,
+        args: codemodArgs,
+        additionalHelpOutput: parser.usage,
+      );
+      if (exitCode != 0) return exitCode;
+    }
+
+    return 0;
+  }
+
+  final FileSystem fs = const LocalFileSystem();
+  final dartPaths = dartFilesToMigrate().toList();
+  // Work around parts being unresolved if you resolve them before their libraries.
+  // TODO - reference analyzer issue for this once it's created
+  final packageRoots = dartPaths.map(findPackageRootFor).toSet().toList();
+  packageRoots.sort((packageA, packageB) =>
+      packageB.split('/').length - packageA.split('/').length);
+
+  Map<String, String> packageNameLookup = Map.fromIterable(
+    pubspecYamlPaths(),
+    key: (path) => findPackageRootFor(path).split('/').last,
+    value: (path) {
+      final pubspec = fs.file(path);
+      List<String> pubspecLines = pubspec.readAsLinesSync();
+      String nameLine =
+          pubspecLines.firstWhere((line) => line.startsWith('name'));
+      var nameLineParts = nameLine.split(':');
+      return nameLineParts[1].trim();
```

Comment:
manually reading as lines and extracting the name could be replaced by using the yaml package and parsing using a library like pubspec, pubspec_parse, etc

Reply:
Yeah, but is it worth pulling in additional deps for 4 lines of code?